mlc_llm/build.py

```diff
@@ -449,6 +450,75 @@ def main():
         print("Reuse existing prebuilt lib {ARGS.reuse_lib}...")
     dump_default_mlc_chat_config(args)
 
+def build_model(model: str=None,
+          hf_path: str=None,
```

Comment:
we should put these into a structured dataclass, otherwise it is really hard to use an interface with the related lists

Reply:
Just for clarification, are we making a [dataclass](https://docs.python.org/3/library/dataclasses.html) that has all the arguments here, meaning that the users need to create this class before they call the interface? Thanks! 

cc: @sbelcmu

Reply:
i think that would be better (as long as it comes with good default), we can further iterate the interface to be more high level later